decentraland/unity-renderer
# style: fix avatar creation background width

This PR was created to fix the avatar creation background, which is not scaling properly depending the screen size. In consequence, the slots are not inside the darker container.

<img width="1104" alt="Screenshot 2023-11-14 at 11 08 56" src="https://github.com/decentraland/unity-renderer/assets/51088292/42167e5c-e53c-43fc-ad25-9e173bd080ac">

### How to test?
1- Open this [link](https://play.decentraland.org/?explorer-branch=style/SignUpBackpackBackground) with a new account to start the sign up flow.
2- In the avatar creation step, change the window size and check that the darker background is scaling properly and the slots columns are inside it, now overlapping its edge.
